cartography/data/indexes.cypher

```diff
@@ -177,3 +177,4 @@ CREATE INDEX ON :AzureVirtualMachine(id);
 CREATE INDEX ON :AzureDataDisk(id);
 CREATE INDEX ON :AzureDisk(id);
 CREATE INDEX ON :AzureSnapshot(id);
+CREATE INDEX ON :AzureAKS(id);
```

Comment:
We should call this AzureCluster. Since the resource we are loading is a Cluster.